src/app/campaign/campaign.component.ts

```diff
@@ -52,19 +53,19 @@ export class CampaignComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit() {
-    this.routeSub = this.route.paramMap.pipe(
-      switchMap((params: ParamMap) => {
-        return this.advertiserService.loadAdvertisers().pipe(
-          map(advertisers => ({params, advertisers}))
-        );
-      })
-    ).subscribe(({params, advertisers}) => {
-      this.campaignStoreService.load(params.get('id'));
-    });
+    this.routeSub = this.route.paramMap
```

Comment:
why the formatting changes? are we not using the same prettier config?

Reply:
We should be, since the `.prettierrc` is committed to Git. A couple of possibilities (assuming you're using VSCode):

- Your formatter isn't always running. I've enabled format on save in the global settings, which does the trick.
<img width="426" alt="Screen Shot 2019-10-17 at 10 59 49 AM" src="https://user-images.githubusercontent.com/1444918/67027036-81049080-f0ce-11e9-9e2f-27583056b18e.png">

- You don't have the Prettier extension installed, or have a formatter other than Prettier set as the default. You can check/change this by opening up the command palette within a TS file and looking at the options under "Format Document With ..."

<img width="312" alt="Screen Shot 2019-10-17 at 11 10 25 AM" src="https://user-images.githubusercontent.com/1444918/67027239-d3de4800-f0ce-11e9-960c-db8e52fbfb66.png">